.travis.yml

```diff
@@ -132,6 +132,10 @@ jobs:
   - name: unit-test
     script:
     - cargo test --all --lib --tests --benches
+
+  # Doc tests.
+  - name: doc-test
```

Comment:
Good idea, although inability to share build cache among this job and unit tests can increase build time for this job (but, hopefully, not the overall build time).

Reply:
Additionally, we use `sscache` for caching which stores its cache to `$HOME/.cache` and this directory shares between jobs. I suppose. By the way. I try to remove .cargo caching. I think in our case it works not optimal and consumes more time for archiving/unarchiving.